test/e2e/organization_test.go

```diff
@@ -0,0 +1,142 @@
+package e2e
+
+import (
+	"io"
+	"os"
+	"regexp"
+	"testing"
+
+	root "github.com/equinix/metal-cli/internal/cli"
+	"github.com/equinix/metal-cli/internal/organizations"
+	outputPkg "github.com/equinix/metal-cli/internal/outputs"
+	"github.com/spf13/cobra"
+)
+
+func TestCli_Organization(t *testing.T) {
+	subCommand := "organization"
+	consumerToken := ""
+	apiURL := ""
+	Version := "metal"
+	rootClient := root.NewClient(consumerToken, apiURL, Version)
+	type fields struct {
+		MainCmd  *cobra.Command
+		Outputer outputPkg.Outputer
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		want    *cobra.Command
+		cmdFunc func(*testing.T, *cobra.Command)
+	}{
+		{
+			name: "get",
+			fields: fields{
+				MainCmd:  organizations.NewClient(rootClient, outputPkg.Outputer(&outputPkg.Standard{})).NewCommand(),
+				Outputer: outputPkg.Outputer(&outputPkg.Standard{}),
+			},
+			want: &cobra.Command{},
+			cmdFunc: func(t *testing.T, c *cobra.Command) {
+				t.Helper()
+				root := c.Root()
+				root.SetArgs([]string{subCommand, "get"})
+				rescueStdout := os.Stdout
+				r, w, _ := os.Pipe()
+				os.Stdout = w
+				if err := root.Execute(); err != nil {
+					t.Error(err)
+				}
+				w.Close()
+				out, _ := io.ReadAll(r)
+				os.Stdout = rescueStdout
+				idPattern := `(\b[0-9a-fA-F]{8}(-[0-9a-fA-F]{4}){3}-[0-9a-fA-F]{12}\b)`
+				matches := regexp.MustCompile(idPattern).FindAllStringSubmatch(string(out[:]), -1)
+				if len(matches) > 0 {
+					id := matches[0][1]
+					t.Log("ORG ID: ", id)
+					err := os.Setenv("METAL_TEST_ORGID", id)
+					if err != nil {
+						t.Errorf("Error setting environment variable: %v ", err)
+					}
+				} else {
+					t.Errorf("No ID found")
+				}
+			},
+		},
+		{
+			name: "update",
```

Comment:
Since org creation doesn't work in metal-cli at the moment, let's remove these `update` and `payment-method` tests so that we're only testing `get`.  We can come back and fill in tests later as part of fixing org creation.

Reply:
updated